modules/router/src/test/collateral.spec.ts

```diff
@@ -771,5 +771,48 @@ describe(testName, () => {
         expect(node.reconcileDeposit.callCount).to.be.eq(2);
       });
     });
+
+    it("should collateralize difference between owed and target", async () => {
+      const { channel } = createTestChannelState(UpdateType.deposit, {
+        aliceIdentifier: routerPublicIdentifier,
+        processedDepositsA: ["0"],
+        assetIds: [AddressZero],
+        balances: [{ amount: ["0", "0"], to: [AddressZero, AddressZero] }],
```

Comment:
the `to` in the balance here should be the channel participants, not the `assetId`